boltdesignsystem/bolt
# Action Blocks | FE | Add spacing prop

## Jira

http://vjira2:8080/browse/BDS-2082

## Summary

Adds a `spacing` prop for the Action Blocks component.

## Details

1. CSS and twig updated.
2. Defaults to `medium` which is the current setting, so no visual breaking changes.

## How to test

Run the branch locally and check the spacing variations page. Make sure the spacing matches what's stated.
